cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py

```diff
@@ -463,13 +465,12 @@ def configure_worker_services(api_servers, dns, cluster_cidr):
     kubelet_opts.add('require-kubeconfig', 'true')
     kubelet_opts.add('kubeconfig', kubeconfig_path)
     kubelet_opts.add('network-plugin', 'cni')
-    kubelet_opts.add('logtostderr', 'true')
     kubelet_opts.add('v', '0')
     kubelet_opts.add('address', '0.0.0.0')
     kubelet_opts.add('port', '10250')
     kubelet_opts.add('cluster-dns', dns['sdn-ip'])
     kubelet_opts.add('cluster-domain', dns['domain'])
-    kubelet_opts.add('anonymous-auth', 'false')
+    kubelet_opts.add('anonymous-auth=false', None)
```

Comment:
What happened here? If `kubelet_opts.add('anonymous-auth', 'false')` didn't work then we should probably fix that in FlagManager rather than work around it here.